src/Symfony/Component/VarDumper/Caster/SymfonyCaster.php

```diff
@@ -66,4 +68,30 @@ public static function castHttpClientResponse($response, array $a, Stub $stub, b
 
         return $a;
     }
+
+    public static function castUuid(Uuid $uuid, array $a, Stub $stub, bool $isNested)
+    {
+        $a[Caster::PREFIX_VIRTUAL.'base 58'] = $uuid->toBase58();
+        $a[Caster::PREFIX_VIRTUAL.'base 32'] = $uuid->toBase32();
+
+        // symfony/uid >= 5.3
+        if (method_exists($uuid, 'getDateTime')) {
+            $a[Caster::PREFIX_VIRTUAL.'timestamp'] = $uuid->getDateTime()->format('Y-m-d H:i:s.u');
+        }
+
+        return $a;
+    }
+
+    public static function castUlid(Ulid $ulid, array $a, Stub $stub, bool $isNested)
+    {
+        $a[Caster::PREFIX_VIRTUAL.'base 58'] = $ulid->toBase58();
+        $a[Caster::PREFIX_VIRTUAL.'RFC 4122'] = $ulid->toRfc4122();
```

Comment:
```suggestion
        $a[Caster::PREFIX_VIRTUAL.'rfc4122'] = $ulid->toRfc4122();
```

What about lowercase instead like for base58 and base32?

Note:
I don't know what other casters use!

Reply:
@nicolas-grekas what about this one?

Reply:
I would prefer `RFC 4122`. That's the usual notation for RFC and after all, the output is meant to be human-readable.

Reply:
I prefer RFC 4122 too.

Reply:
I thought I had posted a comment. My preference would be to use the name of the methods.

Reply:
And we would do the same in the inspect commands?

Reply:
that'd make sense yes!